deployments/charts/kuma/values.yaml

```diff
@@ -225,6 +225,34 @@ ingress:
   # -- Affinity placement rule for the Kuma Ingress pods
   affinity: {}
 
+egress:
+  # -- If true, it deploys Egress for cross cluster communication
+  enabled: false
+  # -- Time for which old listener will still be active as draining
+  drainTime: 30s
+  # -- Number of replicas of the Egress
+  replicas: 1
+  service:
+    # -- Service type of the Egress
+    type: LoadBalancer
+    # -- (string) Optionally specify IP to be used by cloud provider when configuring load balancer
+    loadBalancerIP:
```

Comment:
probably should be removed

Reply:
I would leave it as we have option for NodePort, so maybe somehow, somebody will like to have it as a LoadBalancer (don't know why, but who knows)

Reply:
are you ok with it?